client/components/seo/style.scss

```diff
@@ -104,70 +107,65 @@
 	font-size: 11px;
 	line-height: 11px;
 	text-transform: uppercase;
-	overflow: hidden;
 	text-overflow: ellipsis;
 	white-space: nowrap;
 }
 
-.facebook-opengraph-preview__article
-.facebook-opengraph-preview__image img {
-	height: auto;
-	width: 100%;
-}
-
-.facebook-opengraph-preview__article
-.facebook-opengraph-preview__body {
-	height: auto;
-	max-height: 100px;
-}
-
-.facebook-opengraph-preview__article
-.facebook-opengraph-preview__title {
-	margin-bottom: 12px;
-}
+.facebook-opengraph-preview__article {
+	.facebook-opengraph-preview__content {
+		flex-direction: column;
+		min-width: 100%;
+	}
 
-.facebook-opengraph-preview__article
-.facebook-opengraph-preview__description {
+	.facebook-opengraph-preview__image {
```

Comment:
Oh, I didn't realize I broke the guidelines. I think it gives it a way better structure for readability... happy to switch it back.